index.html

```diff
@@ -26,7 +26,7 @@
       <header>
         <h1 class="header-title">I’m Polina Glad to see you!</h1>
         <p>
-          I’m a software developer! I can help you build a product , feature or
+          I’m a software developer! I can help you build a product, feature or
           website Look through some of my work and experience! If you like what
           you see and have a project you need coded, don’t hestiate to contact
           me.
```

Comment:
- [ ] Each icon must have a link (`a` tag)
- [ ] The button (Let's connect), should be a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design)

Reply:
On the example the let's connect refers to the icons underneath as it wouldn't make sense to put an anchor tag to the let's connect